processing/src/main/java/io/druid/query/QueryDimensionInfo.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query;
+
+import io.druid.query.dimension.DimensionSpec;
+import io.druid.segment.DimensionQueryHelper;
+
+public class QueryDimensionInfo
+{
+  public final DimensionSpec spec;
+  public final DimensionQueryHelper queryHelper;
+  public final String name;
+  public final String outputName;
+  public final Object selector;
```

Comment:
created a "ColumnValueSelector" parent class for DimensionSelector, LongColumnSelector, etc., using that as the parameter type instead now